src/main/java/com/jcabi/jdbc/JdbcSession.java

```diff
@@ -309,6 +310,25 @@ public void commit() throws SQLException {
     }
 
     /**
+     * Call an SQL stored procedure.
+     *
+     * <p>JDBC connection is opened and, optionally, closed by this method.
```

Comment:
@mkordas Yes. In ``run``'s ``finally`` block the ``statement`` is closed, not the connection (I made a mistake in the first comment). ``optionally`` means if it's ``autocommit`` or not... "closing" the connection actually means "commiting" it. And autocommit is dictated by class boolean ``this.auto`` which can be set with the ``autocommit(boolean)`` method. The Javadoc actually has in regards the fluency of the API, that's why it seems a bit hard to understand, in my opinion.